src/develop/ui/patterns/mobile/rangeslider.md

```diff
@@ -3,74 +3,74 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# RangeSlider Pattern
+# Range Slider 
 
-The RangeSlider pattern allows you to set a value by dragging a handle within a configured range. It is used to control a variable value with simple and interactive user input.
+You can use the Range Slider UI Pattern to allow users select a single value between two range values. This pattern enables the adjustment of content within a predetermined range. Moving the slider along the track, increases or decreases the value.  
```

Comment:
I don't understand what "This pattern enables the adjustment of content within a predetermined range." - maybe it's redundant?